corehq/apps/hqmedia/views.py

```diff
@@ -168,6 +169,12 @@ def mime_type(self):
         except Exception as e:
             raise BadMediaFileException("There was an error fetching the MIME type of your file. Error: %s" % e)
 
+    @method_decorator(require_permission(Permissions.edit_apps, login_decorator=login_with_permission_from_post()))
+    # YUI js uploader library doesn't support csrf
```

Comment:
Since we are using YUI flash uploader which doesn't support csrf (I finally understood why flash is not secure) all YUI interfacing views should be csrf exempted. @biyeun is there any way to pass csrf request header in YUI flash uploader request?